### PR TITLE
Fix: APP-2418 - Edit Settings fixes

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Fixes Edit Settings issues for Multisig and MajorityVoting DAOs

Task: [APP-2418](https://aragonassociation.atlassian.net/browse/APP-2418) & [APP-2421](https://aragonassociation.atlassian.net/browse/APP-2421)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2418]: https://aragonassociation.atlassian.net/browse/APP-2418?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[APP-2421]: https://aragonassociation.atlassian.net/browse/APP-2421?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/editSettings/majorityVoting.tsx [hunk 1]
```diff
@@ -185,9 +191,36 @@ export const EditMvSettings: React.FC<EditMvSettingsProps> = ({daoDetails}) => {
     earlyExecution !== daoVotingMode.earlyExecution ||
     voteReplacement !== daoVotingMode.voteReplacement;
 
+  // calculate proposer
+  let daoEligibleProposer: TokenVotingProposalEligibility =
+    formEligibleProposer;
+  if (
+    // TODO: fix dao settings hook
```
Comment (cgero-eth) [inline src/containers/editSettings/majorityVoting.tsx:198]: Is this TODO still valid or can it be removed?

**Review by cgero-eth — COMMENTED** [review]
Thanks @Fabricevladimir!

File: src/containers/editSettings/majorityVoting.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply src/containers/editSettings/majorityVoting.tsx:198]: Still valid unfortunately. There's a ticket for it though, and I'm going to pick it up after this is merged.

Comment (cgero-eth) [reply src/containers/editSettings/majorityVoting.tsx:198]: Okay thanks for the info :)

**Review by cgero-eth — APPROVED** [review]

**Pull request merged**